.travis.yml

```diff
@@ -7,7 +7,7 @@ matrix:
     - rvm: 2.3.4
     - rvm: 2.2
     - rvm: 2.1
-    - rvm: jruby-9.1.8.0
+    - rvm: jruby-9.1.10.0
       env:
```

Comment:
Right, that is unfortunate. Anyway, thanks for the PR, looks good to me! Possibly we can keep track of `jruby` head if it adds value.